Server and/or game crashing when GTceuM or Create Liquid Fuel is present

### What happened?

Forge 47.4.0 on 1.20.1 (No Sinytra is not causing the error.)
Mod Ver: ait-1.2.3.2810-1.20.1-release.jar

<details>

<summary>Server/game fails to boot with this error:</summary>

```
[07May2025 19:08:05.636] [main/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Caught exception during event net.minecraftforge.event.AddPackFindersEvent@1bcf0df6 dispatch for modid gtceu
java.lang.NullPointerException: Cannot invoke "dev.amble.ait.core.drinks.Drink.id()" because "drink" is null
	at dev.amble.ait.core.drinks.DrinkUtil.setDrink(DrinkUtil.java:130) ~[ait-1.2.3.2810-1.20.1-release_mapped_srg_1.20.1.jar%232379!/:?]
	at dev.amble.ait.core.item.DrinkItem.m_7968_(DrinkItem.java:55) ~[ait-1.2.3.2810-1.20.1-release_mapped_srg_1.20.1.jar%232379!/:?]
	at com.gregtechceu.gtceu.utils.GTUtil.getItemBurnTime(GTUtil.java:341) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.data.recipe.misc.FuelRecipes.init(FuelRecipes.java:37) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.common.data.GTRecipes.recipeAddition(GTRecipes.java:65) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.common.CommonProxy.registerPackFinders(CommonProxy.java:274) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.common.__CommonProxy_registerPackFinders_AddPackFindersEvent.invoke(.dynamic) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%2352!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2352!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2352!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:121) ~[javafmllanguage-1.20.1-47.4.0.jar%231249!/:?]
	at net.minecraftforge.fml.ModLoader.lambda$postEvent$29(ModLoader.java:326) ~[fmlcore-1.20.1-47.4.0.jar%231248!/:?]
	at java.lang.Iterable.forEach(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.ModList.forEachModInOrder(ModList.java:227) ~[fmlcore-1.20.1-47.4.0.jar%231248!/:?]
	at net.minecraftforge.fml.ModLoader.postEvent(ModLoader.java:326) ~[fmlcore-1.20.1-47.4.0.jar%231248!/:?]
	at net.minecraft.server.packs.repository.ServerPacksSource.m_245786_(ServerPacksSource.java:52) ~[server-1.20.1-20230612.114412-srg.jar%231247!/:?]
	at net.minecraft.server.Main.main(Main.java:161) ~[server-1.20.1-20230612.114412-srg.jar%231247!/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.4.0.jar%2369!/:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103) ~[fmlloader-1.20.1-47.4.0.jar%2369!/:?]
	at net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27) ~[fmlloader-1.20.1-47.4.0.jar%2369!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?]
[07May2025 19:08:05.636] [main/ERROR] [net.minecraft.server.Main/FATAL]: Failed to start the minecraft server
net.minecraftforge.fml.ModLoadingException: GregTech (gtceu) encountered an error during the done event phase
§7java.lang.NullPointerException: Cannot invoke "dev.amble.ait.core.drinks.Drink.id()" because "drink" is null
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:125) ~[javafmllanguage-1.20.1-47.4.0.jar%231249!/:?]
	at net.minecraftforge.fml.ModLoader.lambda$postEvent$29(ModLoader.java:326) ~[fmlcore-1.20.1-47.4.0.jar%231248!/:?]
	at java.lang.Iterable.forEach(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.ModList.forEachModInOrder(ModList.java:227) ~[fmlcore-1.20.1-47.4.0.jar%231248!/:?]
	at net.minecraftforge.fml.ModLoader.postEvent(ModLoader.java:326) ~[fmlcore-1.20.1-47.4.0.jar%231248!/:?]
	at net.minecraft.server.packs.repository.ServerPacksSource.m_245786_(ServerPacksSource.java:52) ~[server-1.20.1-20230612.114412-srg.jar%231247!/:?]
	at net.minecraft.server.Main.main(Main.java:161) ~[server-1.20.1-20230612.114412-srg.jar%231247!/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.4.0.jar%2369!/:?]
	at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103) ~[fmlloader-1.20.1-47.4.0.jar%2369!/:?]
	at net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27) ~[fmlloader-1.20.1-47.4.0.jar%2369!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar%2355!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "dev.amble.ait.core.drinks.Drink.id()" because "drink" is null
	at dev.amble.ait.core.drinks.DrinkUtil.setDrink(DrinkUtil.java:130) ~[ait-1.2.3.2810-1.20.1-release_mapped_srg_1.20.1.jar%232379!/:?]
	at dev.amble.ait.core.item.DrinkItem.m_7968_(DrinkItem.java:55) ~[ait-1.2.3.2810-1.20.1-release_mapped_srg_1.20.1.jar%232379!/:?]
	at com.gregtechceu.gtceu.utils.GTUtil.getItemBurnTime(GTUtil.java:341) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.data.recipe.misc.FuelRecipes.init(FuelRecipes.java:37) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.common.data.GTRecipes.recipeAddition(GTRecipes.java:65) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.common.CommonProxy.registerPackFinders(CommonProxy.java:274) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at com.gregtechceu.gtceu.common.__CommonProxy_registerPackFinders_AddPackFindersEvent.invoke(.dynamic) ~[gtceu-1.20.1-1.6.4.jar%23934!/:?]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%2352!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%2352!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%2352!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:121) ~[javafmllanguage-1.20.1-47.4.0.jar%231249!/:?]
	... 19 more
``` 
</details>
We removed Create Liquid Fuel because we didn't realise it was still on the server so the error doesn't show that but both mods are trying to get an id for a drink which is returning null and causing the game to fail. Have tested without both mods and the game loads fine.


<details>

<summary>Mod List:</summary>

```
[07May2025 19:07:05.561] [main/INFO] [Puzzles Lib/]: Loading 667 mods:
	- aaa_particles 1.20.1-1.4.5
	- accessories 1.0.0-beta46+1.20.1
		|-- fabric_entity_events_v1 1.6.0+6274ab9d77
		\-- fabric_object_builder_api_v1 11.1.3+2174fc8477
	- ad_astra 1.15.19
	- ad_astra_giselle_addon 6.18
	- ad_astra_rocketed 1.0.3
	- additionalplacements 2.2.2
	- addonslib 1.20.1-1.4
	- adorabuild_structures 2.8.0
	- advanced_ae 1.1.2-1.20.1
	- ae2 15.4.2
	- ae2_mega_things 1.0.8
	- ae2_toggleable_view_cell 1.0.0
	- ae2insertexportcard 1.20.1-1.3.0
	- ae2netanalyser 1.20-1.0.6-forge
	- ae2qolrecipes 1.3.0
	- ae2things 1.2.1
	- ae2wtlib 15.2.3-forge
	- aecapfix 0.9
	- aeinfinitybooster 1.20.1-1.0.0+20
	- aeroblender 1.20.1-1.0.1-neoforge
	- aether 1.20.1-1.5.2-neoforge
		|-- cumulus_menus 1.20.1-1.0.1-neoforge
		\-- nitrogen_internals 1.20.1-1.0.12-neoforge
	- aether_enhanced_extinguishing 1.20.1-1.0.0-neoforge
	- aether_protect_your_moa 1.20.1-1.0.0-neoforge
	- aether_redux 2.0.18
	- aether_villages 1.0.7
	- ait 1.2.3.2810-1.20.1-release
	- alexsmobs 1.22.9
	- alexsmobsnaturalistcompat 1.2.1
	- alltheleaks 0.1.4-beta+1.20.1-forge
		\-- mixinsquared 0.2.0
	- allurement 4.0.0
	- almanac 1.0.2
	- almostunified 1.20.1-0.9.5
	- alternate_current 1.7.0
	- ambientsounds 6.1.9
	- amblekit 1.1.8.93-1.20.1-beta
	- amendments 1.20-1.2.19
	- ancient_aether 0.9.10
	- another_furniture 1.20.1-3.0.1
	- appbot 1.5.0
	- appflux 1.20-1.2.2-forge
	- appleskin 2.5.1+mc1.20.1
	- appliedcooking 4.0.0
	- appmek 1.4.2
	- aquatictorches 1.1.1
	- architectury 9.2.14
	- artifacts 9.5.16
		\-- expandability 9.0.4
	- ascended_quark 1.1.6
	- ash_api 3.0.2+1.20.1
	- atmospheric 6.1.1
	- attributefix 21.0.4
	- audreys_additions 1.20.1-1.0.0
	- automobility 0.4.2+1.20.1-forge
	- autumnity 5.0.2
	- awt 2.0.0
	- azurelib 2.0.41
	- badmeteoroid 1.3-1.20.1
	- ballistix 1.20.1-0.7.1-3
	- balm 7.3.27
		\-- kuma_api 20.1.10
	- barteringstation 8.0.0
	- bclib 3.0.14
	- beautify 2.0.2
	- bellsandwhistles 0.4.3-1.20.x
	- berry_good 7.0.0
	- betterchunkloading 1.20.1-5.4
	- betterdays 1.20.1-3.1.1.1-FORGE
	- betterdeserttemples 1.20-Forge-3.0.3
	- betterdungeons 1.20-Forge-4.0.4
	- betterend 4.0.11
	- betterendisland 1.20-Forge-2.0.6
	- betterfortresses 1.20-Forge-2.0.6
	- betterjungletemples 1.20-Forge-2.0.5
	- bettermineshafts 1.20-Forge-4.0.4
	- betternether 9.0.10
	- betteroceanmonuments 1.20-Forge-3.0.4
	- betterp2p 1.5.0
	- bettertridents 8.0.1
	- betterwitchhuts 1.20-Forge-3.0.3
	- bhc 1.20.1-1.1.0
	- bigger_ae2 1.3.6
	- biomesoplenty 19.0.0.96
	- biomeswevegone 1.5.11
	- black_archive 1.5.3
	- blastcraft 1.20.1-0.4.0-3
	- blockui 1.20.1-1.0.190-snapshot
	- blossom 1.0.7
	- blueprint 7.1.3
	- blueprintfix 1.0.0
	- boatbreakfix 1.0.2
	- boatload 5.0.1
	- bookshelf 20.2.13
	- bosses_of_mass_destruction 1.1.2
	- botania 1.20.1-448-FORGE
	- botanypots 13.0.41
	- botanypotstiers 6.0.1
	- botanytrees 9.0.18
	- botarium 2.3.4
	- bowinfinityfix 2.6.0
	- buildinggadgets2 1.0.7
	- bygonenether 1.3.2
	- byzantine 32
	- caelus 3.2.0+1.20.1
	- camocreepers 2.1.0
	- carpeted 1.20-1.4
	- carryon 2.1.2.7
	- cataclysm 2.65
	- cave_dweller 1.7.0
	- caverns_and_chasms 2.0.0
	- cb_multipart 3.3.0.146
		|-- cb_microblock 3.3.0.146
		\-- cb_multipart_minecraft 3.3.0.146
	- cbc_at 0.0.1-1.20.1-a
	- cccbridge 1.6.3-forge
	- ceilingtorch 1.32
	- cerbons_api 1.1.0
	- charmofundying 6.5.0+1.20.1
	- charms 2.0.1-1.20.1
	- chat_heads 0.13.18
	- chemicalscience 0.1.0-1.20.1
	- chickenchunks 2.10.0.100
	- chipped 3.0.7
	- chunkactivitytracker 1.0.1
	- chunksending 1.20.1-2.8
	- chunky 1.3.146
	- chunkyautopause 0.0.2
	- citadel 2.6.1
	- clayworks 3.0.4
	- clickadv 1.20.1-3.8
	- cloth_config 11.1.136
	- clumps 12.0.0.4
	- cobweb 1.0.1
	- codechickenlib 4.4.0.516
	- collective 8.3
	- colony4cc 2.6.5
	- colony_curios 1.0.0
	- colytra 6.2.2+1.20.1
	- comforts 6.4.0+1.20.1
	- compressedblocks 1.6.1-1.20.2
	- computercraft 1.115.0
	- configuration 3.1.0
	- conjurer_illager 1.1.6
	- connectivity 1.20.1-7.1
	- connectorextras 1.11.2+1.20.1
		|-- amecsapi 1.5.3+mc1.20-pre1
		|-- connectorextras_architectury_bridge 1.11.2+1.20.1
		|-- connectorextras_emi_bridge 1.11.2+1.20.1
		|-- connectorextras_energy_bridge 1.11.2+1.20.1
		|-- connectorextras_geckolib_fabric_compat 1.11.2+1.20.1
		|-- connectorextras_jei_bridge 1.11.2+1.20.1
		|-- connectorextras_kubejs_bridge 1.11.2+1.20.1
		|-- connectorextras_modmenu_bridge 1.11.2+1.20.1
			\-- modmenu 7.2.2
		|-- connectorextras_pehkui_bridge 1.11.2+1.20.1
		|-- connectorextras_rei_bridge 1.11.2+1.20.1
		|-- connectorextras_terrablender_bridge 1.11.2+1.20.1
		|-- forgeconfigapiport 8.0.0
		\-- reach_entity_attributes 2.4.0
	- connectormod 1.0.0-beta.46+1.20.1
	- cookingforblockheads 16.0.13
	- copycats 2.2.0+mc.1.20.1-forge
	- corgilib 4.0.3.3
	- coroutil 1.20.1-1.3.7
	- craftingtweaks 18.2.5
	- crashexploitfixer 1.1.0
	- create 0.5.1.j
		\-- flywheel 0.6.11-13
	- create_central_kitchen 1.3.12
		\-- create_dragon_lib 1.4.3
	- create_confectionery 1.1.0
	- create_connected 0.9.4-mc1.20.1
	- create_enchantment_industry 1.2.9.d
	- create_mechanical_spawner 1.20.1-0.1.1.j-55
	- create_new_age 1.1.2
	- create_power_loader 1.5.0-mc1.20.1
	- create_radar 0.1
	- create_sa 2.0.8
	- create_structures 0.1.1
	- create_the_air_wars 1.67
	- create_vibrant_vaults 0.2.0
	- createaddition 1.20.1-1.2.5
	- createaddoncompatibility 0.2.3
	- createappliedkinetics 1.4.0-1.20.1
	- createartifactscompat 1.0.2.1-1.20.1
	- createbigcannons 5.8.2
		\-- ritchiesprojectilelib 2.0.0-dev+mc.1.20.1-forge-build.182
	- createcasing 1.20.1-1.6.2-fix3
	- createcobblestone 1.4.4+forge-1.20.1-95
	- createdeco 2.0.2-1.20.1-forge
	- createdynlight 1.0.2
	- createoreexcavation 1.5.3
	- createutilities 0.3.0+1.20.1
	- creativecore 2.12.32
	- cristellib 1.1.6
	- ctov 3.4.14
	- cucumber 7.0.13
	- cupboard 1.20.1-2.7
	- curios 7.4.0+1.20.1
		\-- cclayer 0.beta.18
	- cutthrough 8.0.2
	- cyclopscore 1.20.0
	- decorative_blocks 4.1.3
	- deep_aether 1.20.1-1.1.5
	- deeperdarker 1.3.3
	- despawntweaks 1.0.0
	- deuf 1.20.1-1.3
	- dimdoors 5.4.4
	- dimensional_ores 1.0.0
	- dimensionalsycnfixes 1.20.1-0.0.1
	- does_it_tick 1.1.4
	- domum_ornamentum 1.20.1-1.0.285-snapshot
	- drawerstooltip 8.0.0
	- dynamic_asset_generator 4.2.2
	- dynamicelectricity 1.20.1-1.1.0-1
	- dynamictrim 1.4.1
	- dynview 2.3
	- easyanvils 8.0.2
	- easymagic 8.0.1
	- edivadlib 2.0.1
	- eldritch_end 0.3.2
	- electrodynamics 1.20.1-0.9.1-2
	- elysium_api 1.1.0
	- elytraslot 6.4.4+1.20.1
	- emi 1.1.22+1.20.1+forge
	- emi_loot 0.7.6+1.20.1+forge
	- emi_ores 1.2+1.20.1+forge
	- endergetic 5.0.1
	- enderio 6.2.10-beta
	- enderstorage 2.11.0.188
	- environmental 4.0.1
	- epherolib 0.1.2
	- etstlib 3.0.0-beta.6fix
	- excavated_variants 3.0.9
		\-- defaultresources 2.0.0
	- expatternprovider 1.20-1.1.11-forge
	- explorations 1.20.1-1.6.1
	- explorerscompass 1.20.1-1.3.3-forge
	- extendedbogeys 0.5.16-1.20.1-0.5.1.f-forge
	- extra_shells 1.1.0
	- fabric_api 0.92.2+1.11.12+1.20.1
	- fabric_language_kotlin 1.13.1kotlin.2.1.10
	- farmingforblockheads 14.0.2
	- fastasyncworldsave 1.20.1-2.4
	- fastbench 8.0.4
	- faster_random 5.1.0
	- fastfurnace 8.0.2
	- fastpaintings 1.20-1.2.7
	- fastsuite 5.1.0
	- featurerecycler 1.0.1
	- ferritecore 6.0.1
	- flib 0.0.14
	- forge 47.4.0
	- formations 1.0.3
	- formationsnether 1.0.5
	- formationsoverworld 1.0.4
	- framework 0.7.12
	- friendsandfoes 3.0.9
	- ftbessentials 2001.2.2
	- ftblibrary 2001.2.8
	- ftbteams 2001.3.1
	- ftbultimine 2001.1.5
	- ftbxmodcompat 2.1.3
	- fulleng 1.7.3-1.20.1
	- fullstackwatchdog 1.0.1+1.19.2-forge
	- fusion 1.2.7+b
	- fzzy_config 0.6.9+1.20.1+forge
	- gallery 1.0.3
	- galosphere 1.20.1-1.4.1
	- gardens_of_the_dead 4.0.1
	- geckolib 4.7.1.2
	- geophilic_reforged 1.2.0
	- glitchcore 0.0.1.1
	- glodium 1.20-1.5-forge
	- goblintraders 1.9.3
	- grassoverhaul 23.10.10
	- gravestone 1.20.1-1.0.24
	- gravestonecurioscompat 2.1.0
	- greg_paintings 1.0.0
	- gtca 1.3
	- gtceu 1.6.4
		\-- ldlib 1.0.34
	- gtnn 1.1.12
	- guideme 20.1.7
	- handcrafted 3.0.6
	- hauntedharvest 1.20-3.2.0
	- hauntfurnace 0.3.3
	- hearth_and_home 1.20.1-2.0.3
	- hearths 1.0.3
	- hit_feedback 1.20.1-1.1.8
	- hugestructureblocks 1.0.9
	- iaf_patcher 0.2
	- iceandfire 2.1.13-1.20.1
	- iceandfire_curios 1.2.0
	- iceberg 1.1.25
	- illagerinvasion 8.0.7
	- immediatelyfast 1.5.0+1.20.4
	- immersive_aircraft 1.2.2+1.20.1
	- immersive_armors 1.7.0+1.20.1
	- immersive_weathering 1.20.1-2.0.5
	- immersivelanterns 1.0.6
	- immersivemessages 1.0.18
	- immersivesnow 1.3.2
	- improvedtrident 2.1
	- incubation 4.0.4
	- insanelib 1.19.0
	- interiors 0.5.6
	- iron_biplane 0.1
	- ironchests 5.0.2
	- itemfilters 2001.1.0-build.53
	- jade 11.13.1+forge
	- jadeaddons 5.3.1+forge
	- jadecolonies 1.4.2
	- jaopca 4.4.14.26
	- jaopcaextras 3.0.0
	- jaopcagtceu 2.1.1.8
	- jei 15.20.0.106
	- journeymap 5.10.3
	- journeymapteams 1.1.0
	- justenoughprofessions 3.0.1
	- k9mod 1.0-1.20.1-forge
	- kambrik 6.1.1+1.20.1
	- kiwi 11.8.31+forge
	- kjscc 1.0.1-beta
	- kleeslabs 15.0.6
	- konkrete 1.8.0
	- kotlinforforge 4.11.0
	- kubejs 2001.6.5-build.16
	- kubejs_create 2001.2.5-build.2
	- kubejs_enderio 1.20.1-0.6.0
	- kubejs_mekanism 2001.1.5.1-build.2
	- leaky 1.20.1-2.1
	- leavesbegone 8.0.0
	- lionfishapi 2.4-Fix
	- lithostitched 1.4
	- lmft 1.0.4+1.20.1
	- lodestone 1.20.1-1.6.4.1
	- logbegone 1.0.8
	- logprot 1.4
	- lolmct 1.2.7
	- longerchathistory 1.3
	- lootintegration_townsandtowers 1
	- lootintegrations 1.20.1-4.4
	- lootintegrations_bygonepillage 1
	- lootintegrations_cataclysm 1
	- lootintegrations_ctov 1
	- lootintegrations_dnt 1
	- lootintegrations_formations 1
	- lootintegrations_moog 1
	- lootintegrations_phillipsruins 1
	- lootintegrations_structory 1
	- lootintegrations_yungs 1
	- lost_aether_content 1.2.3
	- mae2 1.5.0
	- man_of_many_planes 0.2.0+1.20.1
	- mantle 1.11.44
	- mapfrontiers 2.6.0p5
		\-- commonnetworking 1.0.5-1.20.1
	- mca 7.6.3+1.20.1
	- mcwbiomesoplenty 1.20.1-1.2.1
	- mcwbridges 3.0.0
	- mcwbyg 1.20.1-1.2.1
	- mcwdoors 1.1.2
	- mcwfences 1.2.0
	- mcwfurnitures 3.3.0
	- mcwlights 1.1.2
	- mcwpaintings 1.0.5
	- mcwpaths 1.1.0
	- mcwroofs 2.3.2
	- mcwstairs 1.0.1
	- mcwtrpdoors 1.1.4
	- mcwwindows 2.3.0
	- megacells 2.4.6-1.20.1
	- mekanism 10.4.15
	- mekanism_extras 1.20.1-1.4.1
	- mekanismaaa 1.1.0
	- mekanismadditions 10.4.15
	- mekanismcardboardtooltip 1.4.0
	- mekanismcovers 1.3-BETA+1.20
	- mekanismgenerators 10.4.15
	- mekanismtools 10.4.15
	- melody 1.0.2
	- memoryleakfix 1.1.5
	- memorysettings 1.20.1-5.9
	- merequester 1.20.1-1.1.5
	- mes 1.3.4-1.20-forge
	- midnightlib 1.4.2
	- mindfuldarkness 8.0.3
	- minecolonies 1.20.1-1.1.876-snapshot
	- minecolonies_compatibility 2.72
	- minecolonies_tweaks 2.62
	- minecraft 1.20.1
	- mininggadgets 1.15.6
	- mixintracereforged 1.0.0
	- mns 1.0.1-1.20-forge
	- moderately_enough_effect_desc 3.3.0
	- modernfix 5.21.0+mc1.20.1
	- moderntrainparts 0.1.7-forge-mc1.20.1-cr0.5.1.f
	- moonlight 1.20-2.14.0
	- more_compatibility 0.0.1
	- moremekanismprocessing 4.2
	- mowzies_cataclysm 1.2.0
	- mowziesmobs 1.7.2
	- mr_blossom_blade 1.2
	- mr_create_cobblestonedeepslategenerators 1.4.1+1.20.1
	- mr_create_compatbetterendbetternether 1.1
	- mr_create_decoadditions 1.3.2
	- mr_ctov_beautifycompat 2.0
	- mr_ctov_bygcompat 2.0
	- mr_ctov_createstructures 1.0
	- mr_ctov_friendsandfoescompat 2.0
	- mr_ctov_savageandravagecompat 2.0
	- mr_dungeons_andtaverns 3.0.3.f
	- mr_dungeons_andtavernsancientcityoverhaul 1
	- mr_dungeons_andtavernspillageroutpostrework 1.1
	- mr_dungeons_andtavernsstrongholdrework 1
	- mr_terralith_biomesaplings 1.1.0c
	- mr_tidal_towns 1.3.4
	- mru 1.0.4+1.20.1+forge
	- multidim 2.0.0
	- multipiston 1.20-1.2.43-RELEASE
	- mvs 4.1.2-1.20-forge
	- naturalist 5.0pre2
	- naturescompass 1.20.1-1.11.2-forge
	- necronomicon 1.6.0
	- neruina 2.1.1
	- net_objecthunter_exp4j 0.4.8
	- netherchested 8.0.4
		\-- puzzlesapi 8.1.7
	- netherexp 2.3.2
	- netherportalfix 13.0.1
	- nightconfigfixes 8.0.0
	- nochatreports 1.20.1-v2.2.2
	- noisium 2.3.0+mc1.20-1.20.1
	- notenoughcrashes 4.4.7+1.20.1
	- nuclearscience 1.20.1-0.6.1-0
	- octolib 0.5.0.1
	- ohthetreesyoullgrow 1.3.9
	- org_jetbrains_kotlin_kotlin_reflect 2.1.10
	- org_jetbrains_kotlin_kotlin_stdlib 2.1.10
	- org_jetbrains_kotlin_kotlin_stdlib_jdk7 2.1.10
	- org_jetbrains_kotlin_kotlin_stdlib_jdk8 2.1.10
	- org_jetbrains_kotlinx_atomicfu_jvm 0.27.0
	- org_jetbrains_kotlinx_kotlinx_coroutines_core_jvm 1.10.1
	- org_jetbrains_kotlinx_kotlinx_coroutines_jdk8 1.10.1
	- org_jetbrains_kotlinx_kotlinx_datetime_jvm 0.6.1
	- org_jetbrains_kotlinx_kotlinx_io_bytestring_jvm 0.6.0
	- org_jetbrains_kotlinx_kotlinx_io_core_jvm 0.6.0
	- org_jetbrains_kotlinx_kotlinx_serialization_cbor_jvm 1.8.0
	- org_jetbrains_kotlinx_kotlinx_serialization_core_jvm 1.8.0
	- org_jetbrains_kotlinx_kotlinx_serialization_json_jvm 1.8.0
	- packapunch 0.0.5
	- packetfixer 2.0.0
	- palegarden 1.0.7
	- pamhc2crops 1.0.3
	- pamhc2foodcore 1.0.5
	- pamhc2foodextended 0.0NONE
	- pamhc2trees 1.0.2
	- pantographsandwires 1.20.1-alpha-0.0.4
		\-- dragonlib 1.20.1-2.2.24
	- particle_core 0.2.6+1.20.1+forge
		\-- conditional_mixin 0.6.4
	- patchouli 1.20.1-84.1-FORGE
	- paxi 1.20-Forge-4.0
	- pccard 1.0.15
	- peripheralworks 1.5.1
		|-- broccolium 1.0.0
		|-- peripheralium 1.0.0
		\-- tweakium 1.0.0
	- personality 4.1.0
	- pet_cemetery 2.0.0
	- philipsruins 5.3
	- pickletweaks 8.0.5
	- pingwheel 1.10.2
	- placebo 8.6.3
	- playeranimator 1.0.2-rc1+1.20
	- polyeng 0.1.1-1.20.1
	- polymorph 0.49.10+1.20.1
		\-- spectrelib 0.13.17+1.20.1
	- portallinkingcompass 1.0.1+mc1.20
	- potacore 0.2.8-universal
	- prism 1.0.5
	- progressivebosses 4.3.2-mc1.20.1
		\-- mixinextras 0.5.0-beta.5
	- projectred_core 4.21.0
	- projectred_expansion 4.21.0
	- projectred_exploration 4.21.0
	- projectred_fabrication 4.21.0
	- projectred_illumination 4.21.0
	- projectred_integration 4.21.0
	- projectred_transmission 4.21.0
	- puzzleslib 8.1.32
		\-- puzzlesaccessapi 20.1.1
	- quad 1.2.9
	- quark 4.0-462
	- quarkartifacts 1.0.2.2-1.20.1
	- quarkoddities 1.20.1
	- queue 1.0.2-1.20
	- radium 0.12.4+git.26c9d8e
	- raided 0.1.4
	- railways 1.6.7+forge-mc1.20.1
	- rainbowreef 2.3-1.20.1
	- ramcompat 0.1.4
	- rarcompat 0.1.7
	- rechiseled 1.1.6
	- rechiseled_chipped 1.1
	- rechiseledae2 1.0.0
	- rechiseledcreate 1.0.2
	- recipesforcreate 1.1.4
	- redirected 1.0.0
	- relics 0.8.0.9
	- repurposed_structures 7.1.15+1.20.1-forge
	- resourcefulconfig 2.1.3
	- resourcefullib 2.1.29
	- resourcepackoverrides 8.0.3
	- revampedwolf 1.20.1-6.0.2
	- rhino 2001.2.3-build.10
	- savage_and_ravage 6.0.0
	- scheduler 1.0.2
	- sebastrnlib 4.0.0
	- separatedleaves 2.6.0
	- sereneseasons 9.1.0.2
	- sereneseasonsfix 1.1.0.1
	- sereneseasonsphc2crops 1.20.1-1.0.0
	- servercore 1.5.2+1.20.1
	- sharedadvancements 1.20.1-2.2.0
	- simplyswords 1.56.0-1.20.1
	- sliceanddice 3.3.0
	- smallernetherportals 3.9
	- smallships 2.0.0-b1.4
	- smartbrainlib 1.15
	- smarterfarmers 1.20-2.1.2
	- smoothchunk 1.20.1-4.1
	- snad 20.1.0
	- snowrealmagic 10.6.1
		|-- fabric_api_base 0.4.31+ef105b4977
		\-- fabric_renderer_indigo 1.5.2+b5b2da4177
	- snowundertrees 1.4.10
	- sophisticatedbackpacks 3.23.14.1233
	- sophisticatedcore 1.2.52.967
	- soul_fire_d 4.0.11
	- spark 1.10.53
	- sparsestructuresreforged 1.20.1-0.0.1
	- specialised_cells 1.0.0
	- stoneworks 8.0.0
	- stonycliffs 1.1.2
	- storagedrawers 12.9.13
	- structory 1.3.5
	- structory_towers 1.0.7
	- structure_gel 2.16.2
	- structure_layout_optimizer 1.0.10
	- structureessentials 1.20.1-4.7
	- structurize 1.20.1-1.0.770-snapshot
	- stylecolonies 1.13
	- supermartijn642configlib 1.1.8
	- supermartijn642corelib 1.1.18
	- supplementaries 1.20-3.1.16
	- suppsquared 1.20-1.1.18
	- swem 1.5.3
	- t_and_t 0.0NONE
	- tacz 1.1.4-hotfix
	- tacz_fire_control_extension 1.1.3+mc1.20.1+tacz1.1.4
		\-- gunsmithlib 2.0.0+mc1.20.1+tacz1.1.4
	- taczjs 1.3.4-mc1.20.1
	- tacztweaks 2.4.3
	- takesapillage 1.0.3
	- tardis_refined 2.1.5
	- tcompat 1.2.2
	- tconjei 1.4.3
	- tconstruct 3.9.2.37
	- tconstruct_js 1.0.2
	- tectonic 2.4.1
	- tellme 0.9.1
	- temporalapi 1.6.7
	- terrablender 3.0.1.10
	- terrainslabs 1.4.0
	- terralith 2.5.4
	- tesseract 1.0.37
	- tflostblocks 1.20.1-1.5.0
	- tfmg 0.9.3-1.20.1
	- thaumon 2.3.0+1.20.1
	- the_bumblezone 7.7.0+1.20.1-forge
		\-- athena 3.1.1
	- tinkers_advanced 3.0.0-beta.4
	- tinkerslevellingaddon 1.4.2
	- toofast 0.4.3.5
	- toolstats 16.0.9
	- torchmaster 20.1.9
	- toweringtownscape 1.4-1.20.1
	- towntalk 1.1.0
	- tradingpost 8.0.2
	- transparent 8.0.1+1.20.1
	- trashcans 1.0.18b
	- travelersbackpack 9.1.36
	- triadtech 1.0.3+1.20.1
	- twigs 1.20.1-3.1.1
	- twilightaether 1.20.1-1.0.0
	- twilightforest 4.3.2508
	- twilightforestartifacts 1.0.2.2-1.20.1
	- txnilib 1.0.23
		|-- fabric_api_lookup_api_v1 1.6.36+67f9824077
		|-- fabric_biome_api_v1 13.0.13+dc36698e77
		|-- fabric_block_api_v1 1.0.11+0e6cb7f777
		|-- fabric_block_view_api_v2 1.0.1+0767707077
		|-- fabric_blockrenderlayer_v1 1.1.41+1d0da21e77
		|-- fabric_client_tags_api_v1 1.1.2+5d6761b877
		|-- fabric_command_api_v1 1.2.34+f71b366f77
		|-- fabric_command_api_v2 2.2.13+561530ec77
		|-- fabric_content_registries_v0 4.0.11+a670df1e77
		|-- fabric_convention_tags_v1 1.5.5+fa3d1c0177
		|-- fabric_data_attachment_api_v1 1.0.0+30ef839e77
		|-- fabric_data_generation_api_v1 12.3.4+369cb3a477
		|-- fabric_dimensions_v1 2.1.54+8005d10d77
		|-- fabric_events_interaction_v0 0.6.2+0d0bd5a777
		|-- fabric_game_rule_api_v1 1.0.40+683d4da877
		|-- fabric_item_api_v1 2.1.28+4d0bbcfa77
		|-- fabric_item_group_api_v1 4.0.12+c9161c2d77
		|-- fabric_key_binding_api_v1 1.0.37+561530ec77
		|-- fabric_lifecycle_events_v1 2.2.22+afab492177
		|-- fabric_loot_api_v2 1.2.1+eb28f93e77
		|-- fabric_message_api_v1 5.1.9+52cc178c77
		|-- fabric_mining_level_api_v1 2.1.50+561530ec77
		|-- fabric_model_loading_api_v1 1.0.3+6274ab9d77
		|-- fabric_models_v0 0.4.2+7c3892a477
		|-- fabric_networking_api_v1 1.3.11+503a202477
		|-- fabric_particles_v1 1.1.2+78e1ecb877
		|-- fabric_recipe_api_v1 1.0.21+514a076577
		|-- fabric_registry_sync_v0 2.3.3+1c0ea72177
		|-- fabric_renderer_api_v1 3.2.1+cf68abbe77
		|-- fabric_rendering_data_attachment_v1 0.3.37+a6081afc77
		|-- fabric_rendering_fluids_v1 3.0.28+4ac5e37a77
		|-- fabric_rendering_v1 3.0.8+66e9a48f77
		|-- fabric_resource_conditions_api_v1 2.3.8+9e342fc177
		|-- fabric_resource_loader_v0 0.11.10+bcd08ed377
		|-- fabric_screen_api_v1 2.0.8+45a670a577
		|-- fabric_screen_handler_api_v1 1.3.30+561530ec77
		|-- fabric_sound_api_v1 1.0.13+4f23bd8477
		|-- fabric_transfer_api_v1 3.3.5+631c9cd677
		\-- fabric_transitive_access_wideners_v1 4.3.1+1880499877
	- umbral_skies 1.3
	- unifyeverything 1.0.2.9
	- upgrade_aquatic 6.0.3
	- villagersplus 3.1
	- villagesandpillages 1.0.2
	- vintage_kubejs 1.20.1-1.0.0rc-2
	- vintageimprovements 1.20.1-0.2.0.3
	- visuality 2.0.2
	- void_lib 1.1.5
	- voidtotem 3.0.1
	- watching 1.9.2
	- watut 1.20.1-1.2.3
	- waystones 14.1.11
	- weathersettings 1.20.1-2.1
	- weeping_angels 1
	- wither_spawn_fix 1.0.0
	- woodworks 3.0.2
	- worldedit 7.2.15+6463-5ca4dff
	- wunderlib 1.1.5
	- yes_steve_model 2.4.1
	- yet_another_config_lib_v3 3.6.2+1.20.1-forge
	- ysns 2.0.2
	- yungsapi 1.20-Forge-4.0.6
	- yungsbridges 1.20-Forge-4.0.3
	- yungscavebiomes 1.20.1-Forge-2.0.4
	- yungsextras 1.20-Forge-4.0.3
	- yungsmenutweaks 1.20.1-Forge-1.0.2
	- zeta 1.0-30
```
</details>

We think the both mods are trying to get a burntime for the "drink" and falling when it returns null. If this is right and it's on GtceuM/CLF's end tell me and I'll report to them.

### Version

1.2.1

### Build number & branch

_No response_

### Logs & screenshots

[debug (1).log](https://github.com/user-attachments/files/20091687/debug.1.log)

### I've checked the reported issues...

- [x] ...and this issue was not reported before.

--- a/src/main/java/dev/amble/ait/core/item/blueprint/BlueprintItem.java
+++ b/src/main/java/dev/amble/ait/core/item/blueprint/BlueprintItem.java
@@ -27,10 +27,14 @@ public class BlueprintItem extends Item {
 
     @Override
     public ItemStack getDefaultStack() {
-        ItemStack stack = new ItemStack(this);
-        NbtCompound nbt = stack.getOrCreateNbt();
+        ItemStack stack = super.getDefaultStack();
+        BlueprintSchema blueprint = BlueprintRegistry.getInstance().getRandom();
 
-        nbt.putString("Blueprint", BlueprintRegistry.getInstance().getRandom().id().toString());
+        if (blueprint != null) {
+            NbtCompound nbt = stack.getOrCreateNbt();
+            nbt.putString("Blueprint", blueprint.id().toString());
+        }
+
         return stack;
     }
 

--- a/src/main/java/dev/amble/ait/core/item/blueprint/BlueprintRegistry.java
+++ b/src/main/java/dev/amble/ait/core/item/blueprint/BlueprintRegistry.java
@@ -31,7 +31,7 @@ public class BlueprintRegistry extends SimpleDatapackRegistry<BlueprintSchema> {
 
     @Override
     public BlueprintSchema fallback() {
-        throw new IllegalStateException("No fallback blueprint found");
+        return null;
     }
 
     public static BlueprintRegistry getInstance() {

